resources/fixtures/data/colorColumn-categorical.json

```diff
@@ -0,0 +1,95 @@
+{
+  "title": "FIXTURE: color column categorical",
+  "data": {
+    "table": [
+      [
+        "String",
+        "Number",
+        "Number",
+        "String"
+      ],
+      [
+        "Test1",
+        "1",
+        "2.1",
+        "Test2"
+      ],
+      [
+        "Test2",
+        "2",
+        "2.2",
+        "Test2"
+      ],
+      [
+        "Test3",
+        "3",
+        "2.3",
+        "Test1"
+      ],
+      [
+        "Test4",
+        "4",
+        "2.41",
+        "Test1"
+      ],
+      [
+        "Test5",
+        "5",
+        "2.5",
+        "Test2"
+      ],
+      [
+        "Test6",
+        "6",
+        "2.61",
+        "Test1"
+      ]
+    ],
+    "metaData": {
+      "cells": []
+    }
+  },
+  "sources": [],
+  "options": {
+    "hideTableHeader": false,
+    "showTableSearch": false,
+    "cardLayout": false,
+    "cardLayoutIfSmall": false,
+    "minibar": {
+      "invertColors": false,
+      "barColor": {
+        "positive": {
+          "className": "",
+          "colorCode": ""
+        },
+        "negative": {
+          "className": "",
+          "colorCode": ""
+        }
+      },
+      "selectedColumn": null
+    },
+    "colorColumn": {
```

Comment:
@philipkueng do `colorColumn` options have to be present for the tool to work?- Or are they optional. Do you have to do a migration to roll this feature out?